tests/callbacks/fitting/test_fitting_methods.py

```diff
@@ -219,7 +219,7 @@ def test_linear_model(self):
             grad = 3
             y_intercept = 2
 
-            outp = Linear.model().func(x, m=grad, c=y_intercept)
```

Comment:
can we add a unit test for the case which was previously failing that you've now fixed?